slurm-cluster-chart/values.yaml

```
@@ -1,12 +1,52 @@
 slurmImage: ghcr.io/stackhpc/slurm-docker-cluster:7f4d64e
 
-replicas:
-  slurmd: 2
-  login: 1
+login:
+  # Deployment resource name
+  name: login
+  replicas: 1
 
-nfs:
+slurmd:
+  # StatefulSet resource name
+  name: slurmd
+  replicas: 2
+
+slurmctld:
+  # StatefulSet resource name
+  name: slurmctld
+  # NOTE: We don't include a replicas field here because
+  # replicas > 1 for slurmctld needs extra Slurm config
+
+storage:
   mountPath: /home
-  claimName: rook-nfs-pv-claim
+  # The name of a Read-Write-Many StorageClass to use for
+  # the persistent volume which is shared across Slurm nodes
+  # Note: If using the default value then you must set
+  # rooknfs.enabled = true below to ensure that Rook NFS is 
+  # installed on the cluster as a dependency of this Slurm
+  # chart. If you are using a separate RWM StorageClass, then
+  # set rooknfs.enabled = false
+  storageClassName: &storageclassname slurm-rook-nfs
```

After discussion, lets ditch the yaml anchor here and just set a default 
```suggestion
  storageClassName: slurm-rook-nfs
```